force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        if(a == null || b == null) {
+            return null;
+        }
+        return a + b; // Replace null with the variable you used to store the result
```

Comment:
Try to do this with the safe navigation operator https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_SafeNavigationOperator.htm